### PR TITLE
fix: some french translations are not correct

### PR DESCRIPTION
**Pull request by Zharkan**
### What does it do?

Fixes some translation for FR language

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by ronronscelestes — APPROVED** [review]
LGTM, thank you @Zharkan for this addition 🙂

File: packages/core/admin/admin/src/translations/fr.json [hunk 1]
```diff
@@ -588,6 +588,38 @@
...
+  "global.name": "Nom",
+  "global.none": "Aucun",
+  "global.password": "Mot de passe",
+  "global.plugins": "Plugins",
+  "global.profile": "Profil",
+  "global.reset-password": "Réinitialiser le mot de passe",
+  "global.roles": "Rôles",
+  "global.save": "Enregistrer",
+  "global.see-more": "Voir plus",
+  "global.select": "Séléctionner",
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/translations/fr.json:618]: oh sorry I missed this one before approving, could you fix it please @Zharkan? ('_Selectionner_')

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
added a small comment about "Sélectionner" 🙂

File: packages/core/admin/admin/src/translations/fr.json (hunk 1 above)
Comment (Zharkan) [reply packages/core/admin/admin/src/translations/fr.json:618]: My bad, fixed it !

**Review by ronronscelestes — APPROVED** [review]
🎉

**Pull request merged**